### PR TITLE
Update renovate's configuration

### PR DESCRIPTION
**Pull request by gnaponie**
ref: [CWFHEALTH-2950](https://issues.redhat.com//browse/CWFHEALTH-2950)

### CODE REVIEW

**Review by staticf0x — APPROVED** [review]
LGTM

File: components/mintmaker/base/renovate-config.yaml [hunk 1]
```diff
@@ -28,8 +22,31 @@ data:
             "matchDepPatterns": [
               "^quay.io/redhat-appstudio-tekton-catalog/"
             ],
-            "semanticCommits": "enabled"
+            "groupName": "RHTAP references",
```
Comment (mmorhun) [inline components/mintmaker/base/renovate-config.yaml:25]: Probably we need `Konflux`

File: components/mintmaker/base/renovate-config.yaml [hunk 2]
```diff
@@ -28,8 +22,31 @@ data:
...
+            "branchName": "konflux/references/{{baseBranch}}",
+            "commitBody": "Signed-off-by: {{{gitAuthor}}}",
+            "commitMessageTopic": "RHTAP references",
```
Comment (mmorhun) [inline components/mintmaker/base/renovate-config.yaml:28]: `Konflux`

**Review by mmorhun — APPROVED** [review]

File: components/mintmaker/base/renovate-config.yaml [hunk 3]
```diff
@@ -7,19 +7,13 @@ data:
   renovate.json: |
```
Comment (qixiang) [inline components/mintmaker/base/renovate-config.yaml:7]: Will there be another PR to enable only the three managers later, after we change the Renovate image to our custom image that supports the RPM manager?

Comment (gnaponie) [reply components/mintmaker/base/renovate-config.yaml:7]: No, I just forgot :) I've updated this PR to have only those 3 managers.
BTW I didn't add a preset on purpose, I see our (rebuilds guild) preset might not be ok with this conf. So let's just skip it for now, I don't really see the benefit at this point. We can see if we need a preset in the future.

**Pull request merged**